catalog/opensearch/catalog-opensearch-source/src/main/java/ddf/catalog/source/opensearch/impl/OpenSearchSource.java

```diff
@@ -259,6 +259,7 @@ public void destroy(int code) {
   @Override
   public boolean isAvailable() {
     boolean isAvailable = false;
+    if (!isAvailable) return true;
```

Comment:
Isn't this condition always true?

Reply:
Oops.
I was using this to test and look at the queries as I was building them. Forgot to remove. 😅

Reply:
I figured it had to do with some kind of testing 😄